neqo-http3/src/qlog.rs

```diff
@@ -6,42 +6,69 @@
 
 // Functions that handle capturing QLOG traces.
```

Comment:
I wonder if this (like -transport) could be a lot simpler:

```rust
#[cfg(feature = "qlog")]
mod qlog;
#[cfg(not(feature = "qlog"))]
mod qlog { include!("qlog-stub.rs"); }
```

No need then to have warnings and whatnot, you just have to keep the definitions synchronized.

The only reason you need this is that these functions are exposed to the types in the qlog crate and you don't want to compile the qlog crate when the feature is disabled.